.github/workflows/publish-package-to-npmjs.yml

```diff
@@ -68,6 +68,17 @@ jobs:
           path: ./dist/euid-sdk-${{ steps.version.outputs.package_version }}.js
     outputs:
       sdkVersion: ${{ steps.version.outputs.package_version }}
+  createNpmJsRelease:
+    id: github_release
+    uses: IABTechLab/uid2-shared-actions/actions/shared_create_releases@v2
```

Comment:
This shared action seems to only be for Docker and Maven releases - the main thing this repo is for is the JS SDK. I think the most straight-forward option is to call softprops/action-gh-release directly. Take a look at the last step here: https://github.com/IABTechLab/uid2-shared-actions/blob/v3.20/actions/shared_create_releases/action.yaml

I think if you omit the body parameter you'll get the auto-generated release notes from the included PRs, but I'm not certain - you might need an extra step.